src/headers/string_op.h

```diff
@@ -191,21 +191,11 @@ char * wstr_escape_json(const char * string);
 char * wstr_unescape_json(const char * string);
 
 /**
- * @brief Check if the specified string is alphanumeric 
- * 
- * @param string Input string
- * @return true if the string is alphanumeric.
- */
-bool is_string_alphanumeric(const char* string);
-
-
-/**
- * @brief Convert string to time_t 
- * 
+ * @brief Lowercase a string
```

Comment:
this is adapting to how is the master branch:
https://github.com/wazuh/wazuh/blob/ff962364251c1fa38468bc750c8431ced9254b84/src/shared/string_op.c#L915